src/platform/i386/kernel.c

```diff
@@ -15,6 +15,8 @@
 #define CMDLINE_REQ_LEN (ADDR_STR_LEN * 2 + 1)
 
 struct mem_layout glb_memlayout;
+volatile int cores_ready[NUM_CPU];
+volatile int bsp_ready = 0;
```

Comment:
Perhaps `is_bsp_ready`?

Reply:
Yeah, I will remove bsp_ready, and use cires_ready[0] instead.